_api-reference/cluster-api/shard-stores.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: default 
+title: Shard stores 
+parent: Cluster APIs 
+nav_order: 70
+---
+
+# Shard stores
+
+The `_shard_stores` API provides information about the shard copies for one or more indices. This API helps to diagnose issues with unallocated shards by providing reasons why shards are unassigned and their current states.
+
+<!-- spec_insert_start
+api: indices.shard_stores
+component: endpoints
+-->
+## Endpoints
+```json
+GET /_shard_stores
+GET /{index}/_shard_stores
+```
+<!-- spec_insert_end -->
+
+<!-- spec_insert_start
+api: indices.shard_stores
+component: path_parameters
+include_global: false
+include_deprecated: false
+pretty: false
+-->
+## Path parameters
+
+The following table lists the available path parameters. All path parameters are optional.
+
+| Parameter | Data type | Description |
+| :--- | :--- | :--- |
+| `index` | List or String | List of data streams, indexes, and aliases used to limit the request. |
```

Comment:
```suggestion
| `index` | List or String | Limits health reporting to a specific source. Can be a single source or a comma-separated list of sources (data streams, indexes, or aliases). |
```

Reply:
@kolchfa-aws Mix is allowed, I've updated the api [PR](https://github.com/opensearch-project/opensearch-api-specification/pull/931/commits/88e556fcdea60dada4e0d98178330c74eb1754e2)  with:

"Limits health reporting to a specific source. Can be a single source or a comma-separated list of sources (comprised of data streams, indexes, and aliases)."